pvs/default/props.conf

```diff
@@ -0,0 +1,66 @@
+[source::*]
+TRANSFORMS-changesourcetype = set_sourcetype_nnm, set_sourcetype_pvs
+
+[pvs]
+
+#Example Event
+#Jan 31 16:07:36 192.168.111.18 Jan 31 16:02:09 pvs: 192.168.111.15:53|192.168.111.15:53725|17|7027|DNS Client Failed Query|PVS has observed this host perform a failed DNS lookup for from the DNS server at 192.168.111.1||NONE
+
+#Timestamp Extraction
+MAX_TIMESTAMP_LOOKAHEAD = = 40
+TIME_FORMAT = %b %d %H:%M:%S
+
+#Extract Statement
+EXTRACT-nnm_type,src,src_port,dest,dest_port,protocol,plugin_id,plugin_name,plugin_description,plugin_output,risk = (?P<nnm_type>pvs):\s(?P<src>\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}):(?P<src_port>\d{1,5})\|(?P<dest>\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}):(?P<dest_port>\d{1,5})\|(?P<protocol>\d{1,3})\|(?P<plugin_id>\d{1,5})\|(?P<plugin_name>[^|]+)\|(?P<plugin_description>[^\|]+)\|(?P<plugin_output>.*)?\|(?P<risk>[^\|]+)
+
+
+# Fieldalias
+FIELDALIAS-src = src AS src_ip
+FIELDALIAS-dest = dest AS dest_ip
+
+# Lookups
+LOOKUP-proto = protocol_lookup protocol OUTPUT protocol_name
+LOOKUP-srcport = port_lookup port AS src_port OUTPUT service AS src_svc
+LOOKUP-destport = port_lookup port AS dest_port OUTPUT service AS dest_svc
+
+[tenable:pvs]
```

Comment:
pvs?

Reply:
We have to accept old PVS data inside this app. In the scenario that someone is running old PVS still.  We did some renaming to make it easier to identify these later though with the source type of tenable:pvs rather than just pvs.